webdriver/tests/bidi/script/evaluate/invalid.py

```diff
@@ -0,0 +1,78 @@
+import pytest
+import webdriver.bidi.error as error
+from ... import recursive_compare
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.mark.parametrize("context", [None, False, 42, {}, []])
+async def test_params_context_invalid_type(bidi_session, context):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.evaluate(
+            expression="1 + 2",
+            target=bidi_session.script.ContextTarget(context))
+
+
+async def test_params_context_unknown(bidi_session):
```

Comment:
It's more an invalid value test. So maybe:

```suggestion
async def test_params_context_invalid_value(bidi_session):
```

Mind also adding a test for invalid types (and value) for the `sandbox` argument? And one for not passing in a target at all?

Reply:
Added `test_params_target_invalid_type` and `test_params_sandbox_invalid_type`